tutor/templates/apps/caddy/Caddyfile

```diff
@@ -37,6 +37,8 @@
     reverse_proxy {args.0} {
         header_up X-Forwarded-Port {{ 443 if ENABLE_HTTPS else 80 }}
     }
+
+    {{ patch("caddyfile-reverse-proxy")|indent(4) }}
```

Comment:
Is this reverse-proxy or proxy patch in general? The location of patch is outside the reverse_proxy directive. 
https://caddyserver.com/docs/caddyfile/directives/reverse_proxy

Reply:
@DawoudSheraz Agreed, I have renamed and pushed my changes